<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="hspacer_add_lift">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="l_add">
       <property name="text">
        <string>添加内容</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacer_add_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="sw_detail">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_detail_0_flight">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <widget class="QLabel" name="l_flight_4">
         <property name="text">
          <string>出发地</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="l_flight_0">
         <property name="text">
          <string>航班号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_flight_0">
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_flight_2"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sb_flight_1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_flight_1"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_flight_0"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_flight_2">
         <property name="text">
          <string>座位总数</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="l_flight_5">
         <property name="text">
          <string>到达地</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_flight_1">
         <property name="text">
          <string>价格</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_detail_1_hotel">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_hotel_0"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_hotel_1">
         <property name="text">
          <string>价格</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_hotel_2">
         <property name="text">
          <string>房间总数</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sb_hotel_1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_hotel_0">
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="l_hotel_0">
         <property name="text">
          <string>宾馆地址</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_detail_2_bus">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_bus_0">
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="l_bus_0">
         <property name="text">
          <string>车站地址</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_bus_0"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sb_bus_1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_bus_2">
         <property name="text">
          <string>座位总数</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_bus_1">
         <property name="text">
          <string>价格</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_detail_3_user">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="l_user_0">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_user_0"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_user_1">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_user_1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_user_2">
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_user_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_detail_4_reservation">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QComboBox" name="cb_resv_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_resv_2">
         <property name="text">
          <string>预订内容</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="l_resv_0">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_resv_1">
         <property name="text">
          <string>预订类型</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cb_resv_1">
         <item>
          <property name="text">
           <string>航班</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>宾馆</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>客车</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_resv_0"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
